tests/unit/test_crawlutils.py

```diff
@@ -142,7 +142,7 @@ def test_snapshot_generic_invm_failure(self, *args):
         ("crawler.crawlutils.plugins_manager."
             "get_container_crawl_plugins"),
         side_effect=lambda features: [])
-    def test_snapshot_generic_outcontainer(self, *args):
+    def _test_snapshot_generic_outcontainer(self, *args):
```

Comment:
Ahh, just read the comment about MockedFeaturesCrawler. So, we have to mock the plugins.